main.py

```diff
@@ -20,6 +24,17 @@ def schemas_details():
         return yaml.safe_load(response.content)
 
 
+@functools.lru_cache()
```

Comment:
See [functools](https://docs.python.org/3.3/library/functools.html)

> Decorator to wrap a function with a memoizing callable that saves up to the maxsize most recent calls.

Execute the function call only once